mods/default/models/torch_wall.obj

```diff
@@ -0,0 +1,48 @@
+# Blender v2.73 (sub 0) OBJ File: 'mt_torch_full_vertices3.blend'
+# www.blender.org
+mtllib torch_wall4.mtl
+o Cube_Cube.001
+v 0.061653 -0.554493 -0.328908
+v 0.061653 -0.442208 -0.381835
+v -0.059258 -0.554493 -0.328908
+v -0.059258 -0.442208 -0.381835
+v -0.059994 -0.948766 -0.143618
```

Comment:
Okay... So you rotated the former geometry to keep it simple, i assume. But that vertice and the other face's V16(L20) are practically already in the middle of the node below. Are you sure about this?
Looks kinda odd now.